src/jit/compiler.cpp

```diff
@@ -11091,3 +11091,182 @@ bool Compiler::killGCRefs(GenTree* tree)
 
     return false;
 }
+
+unsigned int Compiler::getAmountLiveRangesReported()
+{
+
+    unsigned int varNum, amountOfLiveRanges = 0;
+    LclVarDsc*   varDsc = lvaTable;
+
+    for (varNum = 0, varDsc = lvaTable; varNum < info.compLocalsCount; varNum++, varDsc++)
+    {
+        if (compMap2ILvarNum(varNum) != (unsigned int)ICorDebugInfo::UNKNOWN_ILNUM)
+        {
+            amountOfLiveRanges += (unsigned int)varDsc->getAmountLiveRanges();
+        }
+    }
+
+    return amountOfLiveRanges;
+}
+
+//------------------------------------------------------------------------
+// startOrCloseVariableLiveRange: Starts or ends a "VariableLiveRange" for "varDsc" if it is
+// borning or dying respectively. Ranges are close-open "[)" so nothing is done in case
```

Comment:
Wording: use "being born" in place of "borning" or "being borning", and "isBorn" in place of "isBorning". See `CodeGenInterface::genUpdateRegLife` in codegencommon.cpp